relationship_server.py

```diff
@@ -11,6 +11,41 @@
 nlp = spacy.load("en_core_web_lg")
 
 logger = logging.getLogger(__name__)
+
+
+'''
+    Utliity method to convert keys and values in a dictionary to lowercase.
+'''
+def makeDictLowercase(d):
+    lowerCaseDict = dict()
+    for k in d.keys():
+        lowerCaseDict[k.lower()] = d[k].lower()
+    return lowerCaseDict
+
+'''
+    Utility to convert string to Title Case
+    Input: this is a sentence.
+    Output: This Is A Sentence.
+'''
+def toTitleCase(string):
```

Comment:
This feels very brittle. 

As an example, what happens in case of "u.k" -- it'd return "U.k" which looks worse? It'd reduce the impact of your PR.

Reply:
> This feels very brittle.
> 
> As an example, what happens in case of "u.k" -- it'd return "U.k" which looks worse? It'd reduce the impact of your PR.

Ideally, a case U.k should be mapped to United Kingdom. But if we are missing an acronym to country mapping in our data set, such a situation might arise.

Do you suggest I keep the country in lowercase itself?

Reply:
Removed the function in the further commits